### PR TITLE
Interactive dataset page

### LINKED ISSUE #420: build new Explore the Data landing page
Please build a new landing page for our "Explore the Data" tab. We would like this to include cards of our different interactive data tools in the style of [Reuters' page for interactives.](http://graphics.reuters.com/), to include these cards: 
- Texas shootings by law enforcement 
- Texas shootings of law enforcement 
- Custodial Deaths in Texas; 
- COVID-19 deaths in Texas incarceration facilities; 
- COVID-19 deaths of Texas officers (still being built); 
- Pre-trial deaths in Texas jails

### PR DESCRIPTION
**Pull request by LayaTaal**
Future landing page for dynamic datasets. Closes #420 

image

### CODE REVIEW

File: components/Card.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+/* eslint-disable react/no-danger */
+
```
Comment (nholden) [inline components/Card.js:2]: Can we dump this? I don't see where we're breaking the `react/no-danger` linting rule in this file.

```suggestion
```

File: pages/interactive.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+/* eslint-disable react/no-danger */
+
```
Comment (nholden) [inline pages/interactive.js:2]:
```suggestion
```

File: components/Card.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                <div
+                  className="card__thumbnail"
+                  style={{ backgroundImage: `url(static/images/thumbnails/${thumbnail})` }}
+                ></div>
+              )}
+              {title && <h3 className="card__title">{title}</h3>}
+              {excerpt && <p className="card__excerpt">{excerpt}</p>}
+            </a>
+          </Link>
+        )}
```
Comment (nholden) [inline components/Card.js:27]: We're already checking `if (link)` above, so I think we can clean this up a bit.

```suggestion
        <Link href={link}>
          <a className="flexwrap">
            {thumbnail && (
              <div
                className="card__thumbnail"
                style={{ backgroundImage: `url(static/images/thumbnails/${thumbnail})` }}
              ></div>
            )}
            {title && <h3 className="card__title">{title}</h3>}
            {excerpt && <p className="card__excerpt">{excerpt}</p>}
          </a>
        </Link>
```

File: package-lock.json [hunk 4]
```diff
@@ -5107,11 +5107,6 @@
         "assert-plus": "^1.0.0"
       }
     },
-    "date-fns": {
-      "version": "2.15.0",
-      "resolved": "https://registry.npmjs.org/date-fns/-/date-fns-2.15.0.tgz",
-      "integrity": "sha512-ZCPzAMJZn3rNUvvQIMlXhDr4A+Ar07eLeGsGREoWU19a3Pqf5oYa+ccd+B3F6XVtQY6HANMFdOQ8A+ipFnvJdQ=="
-    },
```
Comment (nholden) [inline package-lock.json:5114]: Nice cleanup!

File: pages/interactive.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+import React from 'react';
+import Head from 'next/head';
+import styled from 'styled-components';
+import Layout from '../components/Layout';
+import Primary from '../components/Primary';
+import Card from '../components/Card';
+
+const title = 'Dynamic Data sets';
```
Comment (nholden) [inline pages/interactive.js:10]: I like "interactive" from the page name or "interactives" from Reuters. What do you think?

**Review by nholden — COMMENTED** [review]
Yessssssssss! Screenshot looks beautiful. 😍 

I couldn't figure out how to poke at this in the deploy preview. Do we need to add `/interactive` to `next.config.js`?

File: pages/interactive.js (hunk 5 above)
Comment (LayaTaal) [reply pages/interactive.js:8]: I like either of those two, going to leave it as is for now and we can get a consensus at the meeting tomorrow.

**Comment by LayaTaal** [comment]
@nholden thanks for looking at this. The path should work now, you are right that I needed to add it to the pathmap in next.config.

https://deploy-preview-426--texasjusticeinitiative.netlify.app/interactive

**Pull request closed without merging**